app/src/main/java/com/sun/basic_japanese/data/source/local/AppDatabase.kt

```diff
@@ -108,6 +111,30 @@ class AppDatabase private constructor(
         return nhkLessons
     }
 
+    fun getStrokeOrder(input: String): String? {
+        val id = STROKE_ORDER_DEFAULT_ID + input.codePointAt(0).toString(16)
```

Comment:
Nên đặt tên const cho 0 và 16 cho dễ đọc, dễ hiểu.

Reply:
0 nghĩa là kí tự đầu tiên của chuỗi thì nên đặt tên là gì?

Reply:
input.first().toInt().toString()
```suggestion
        val id = STROKE_ORDER_DEFAULT_ID + input.first().toInt().toString()
```